release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1618,6 +1645,18 @@ In this update, `systemd` service only sets the default RPS mask for virtual int
 
 * Previously, the output for the `oc explain router.status.ingress.conditions` command explain route status showed `Currently only Ready` rather than `Admitted` due to incorrect wording in the Application Programming Interface (API). This fix corrects the wording in the API. As a result, the command output is correct. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2041133#[*BZ#2041133*])
 
+* Previously,  the Ingress Operator detected that the user modified an annotation that the Operator manages on the `LoadBalancer-type` service. Consequently, the Operator sets the Ingress Cluster Operator's `Upgradeable` status condition `False` to block upgrades. and the Ingress Operator erroneously set the `Upgradeable` status condition to `False`, blocking upgrades, if the service had no annotations. Fix: The logic that checks the service's annotations was fixed to handle empty annotations correctly. Result: The ingress operator should no longer erroneously block upgrades. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2097555[*BZ#2097555*])
+
+* Previously, the Ingress Operator removed a finalizer that the Operator added to `LoadBalancer-type` services from previous versions of {product-title}. With this update, the Ingress Operator no longer removes finalizers. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2069457[*BZ#2069457*])
+
+* The Ingress Operator performs health checks against the ingress canary route. Berfore this update,  keepalives are enabled on the connection causing the Ingress Operator not to close the TCP Connection to the load balancer (LB) once the health check was completed. A new connection was created for the next health check instead of using the existing connection. Consequently, the connection built on the LB and creating too many connections on the LB. With this update,  keepalives are disabled when connecting to the canary route, and a new connection is made and closed each time the canary probe is run. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2037447[*BZ#2037447*])
```

Comment:
```suggestion
* The Ingress Operator performs health checks against the ingress canary route. Before this update, `keepalive` daemons are enabled on the connection, which caused the Ingress Operator not to close the TCP Connection to the load balancer (LB) after the health check was completed. A new connection was created for the next health check instead of using the existing connection. Consequently, the connection built on the load balancer and creating too many connections on the load balancer. With this update, `keepalive` daemons are disabled when connecting to the canary route, and a new connection is made and closed each time the canary probe is run. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2037447[*BZ#2037447*])
```